BenchBnb/BenchBnb/Models/User.cs

```diff
@@ -8,7 +10,19 @@ namespace BenchBnb.Models
     public class User
     {
         public int Id { get; set; }
-        string UserName { get; set; }
-        string Password { get; set; }
+        [Required, Column("UserName")]
+        public string UserName { get; set; }
+        [Required, Column("Password")]
+        public string Password { get; set; }
```

Comment:
Good use of `Required` attributes here to prevent `null` values, but the `Column` attributes aren't necessary.

Reply:
Fixed